### PR TITLE
Xgboost pyspark integration: User guide doc and tutorials

### PR DESCRIPTION
**Pull request by WeichenXu123**
Xgboost pyspark integration: User guide doc and tutorials

### CODE REVIEW

File: doc/tutorials/spark_estimator.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+###############################
+Using XGBoost PySpark Estimator
+###############################
+Starting from version 1.5, xgboost supports pyspark estimator APIs.
```
Comment (wbo4958) [inline doc/tutorials/spark_estimator.rst:4]: 1.5 -> 2.0 ?

**Comment by wbo4958** [comment]
Overall, LGTM.

File: doc/tutorials/spark_estimator.rst (hunk 1 above)
Comment (WeichenXu123) [reply doc/tutorials/spark_estimator.rst]: Updated.

**Review by wbo4958 — APPROVED** [review]

**Comment by WeichenXu123** [comment]
@wbo4958 Is it good to merge ?

**Comment by wbo4958** [comment]
Yes, I think of, at least LGTM.  @trivialfis could you help to review and merge it?

**Review by trivialfis — APPROVED** [review]

**Pull request merged**